lib/airbrake/logger/logger_ext.rb

```diff
@@ -75,4 +75,20 @@ def airbrake_severity(severity)
        'critical'
      end).freeze
   end
+
+  # @!macro helper_method
+  #   @note We define this helper instead of using instance variable to avoid
+  #     Ruby's warnings about uninitialized ivars.
+
+  ##
+  # @macro helper_method
+  def current_airbrake_notifier
+    airbrake_notifier || Airbrake[:default]
+  end
+
+  ##
+  # @marcro helper_method
+  def current_airbrake_severity
```

Comment:
`current_` prefix is misleading. It is usually used when var changes during method execution, e.g. in iteration. It is also not clear why `airbrake_severity_level` becomes `current_airbrake_severity`.